scripts/clones_analysis/clones_analysis.ipynb

```diff
@@ -0,0 +1,7188 @@
4_col4,#T_f2451_row4_col5,#T_f2451_row5_col0,#T_f2451_row5_col1,#T_f2451_row5_col2,#T_f2451_row5_col3,#T_f2451_row5_col4,#T_f2451_row5_col5,#T_f2451_row6_col0,#T_f2451_row6_col1,#T_f2451_row6_col2,#T_f2451_row6_col3,#T_f2451_row6_col4,#T_f2451_row6_col5,#T_f2451_row7_col0,#T_f2451_row7_col1,#T_f2451_row7_col2,#T_f2451_row7_col3,#T_f2451_row7_col4,#T_f2451_row7_col5,#T_f2451_row8_col0,#T_f2451_row8_col1,#T_f2451_row8_col2,#T_f2451_row8_col3,#T_f2451_row8_col4,#T_f2451_row8_col5,#T_f2451_row9_col0,#T_f2451_row9_col1,#T_f2451_row9_col2,#T_f2451_row9_col3,#T_f2451_row9_col4,#T_f2451_row9_col5,#T_f2451_row10_col0,#T_f2451_row10_col1,#T_f2451_row10_col2,#T_f2451_row10_col3,#T_f2451_row10_col4,#T_f2451_row10_col5,#T_f2451_row11_col0,#T_f2451_row11_col1,#T_f2451_row11_col2,#T_f2451_row11_col3,#T_f2451_row11_col4,#T_f2451_row11_col5,#T_f2451_row12_col0,#T_f2451_row12_col1,#T_f2451_row12_col2,#T_f2451_row12_col3,#T_f2451_row12_col4,#T_f2451_row12_col5,#T_f2451_row13_col0,#T_f2451_row13_col1,#T_f2451_row13_col2,#T_f2451_row13_col3,#T_f2451_row13_col4,#T_f2451_row13_col5,#T_f2451_row14_col0,#T_f2451_row14_col1,#T_f2451_row14_col2,#T_f2451_row14_col3,#T_f2451_row14_col4,#T_f2451_row14_col5,#T_f2451_row15_col0,#T_f2451_row15_col1,#T_f2451_row15_col2,#T_f2451_row15_col3,#T_f2451_row15_col4,#T_f2451_row15_col5,#T_f2451_row16_col0,#T_f2451_row16_col1,#T_f2451_row16_col2,#T_f2451_row16_col3,#T_f2451_row16_col4,#T_f2451_row16_col5,#T_f2451_row17_col0,#T_f2451_row17_col1,#T_f2451_row17_col2,#T_f2451_row17_col3,#T_f2451_row17_col4,#T_f2451_row17_col5,#T_f2451_row18_col0,#T_f2451_row18_col1,#T_f2451_row18_col2,#T_f2451_row18_col3,#T_f2451_row18_col4,#T_f2451_row18_col5,#T_f2451_row19_col0,#T_f2451_row19_col1,#T_f2451_row19_col2,#T_f2451_row19_col3,#T_f2451_row19_col4,#T_f2451_row19_col5,#T_f2451_row20_col0,#T_f2451_row20_col1,#T_f2451_row20_col2,#T_f2451_row20_col3,#T_f2451_row20_col4,#T_f2451_row20_col5,#T_f2451_row21_col0,#T_f2451_row21_col1,#T_f2451_row21_col2,#T_f2451_row21_col3,#T_f2451_row21_col4,#T_f2451_row21_col5,#T_f2451_row22_col0,#T_f2451_row22_col1,#T_f2451_row22_col2,#T_f2451_row22_col3,#T_f2451_row22_col4,#T_f2451_row22_col5,#T_f2451_row23_col0,#T_f2451_row23_col1,#T_f2451_row23_col2,#T_f2451_row23_col3,#T_f2451_row23_col4,#T_f2451_row23_col5,#T_f2451_row24_col0,#T_f2451_row24_col1,#T_f2451_row24_col2,#T_f2451_row24_col3,#T_f2451_row24_col4,#T_f2451_row24_col5,#T_f2451_row25_col0,#T_f2451_row25_col1,#T_f2451_row25_col2,#T_f2451_row25_col3,#T_f2451_row25_col4,#T_f2451_row25_col5,#T_f2451_row26_col0,#T_f2451_row26_col1,#T_f2451_row26_col2,#T_f2451_row26_col3,#T_f2451_row26_col4,#T_f2451_row26_col5,#T_f2451_row27_col0,#T_f2451_row27_col1,#T_f2451_row27_col2,#T_f2451_row27_col3,#T_f2451_row27_col4,#T_f2451_row27_col5,#T_f2451_row28_col0,#T_f2451_row28_col1,#T_f2451_row28_col2,#T_f2451_row28_col3,#T_f2451_row28_col4,#T_f2451_row28_col5,#T_f2451_row29_col0,#T_f2451_row29_col1,#T_f2451_row29_col2,#T_f2451_row29_col3,#T_f2451_row29_col4,#T_f2451_row29_col5{\n",
```

Comment:
Here and further on, once again, please add Markdown descriptions for every table.